extensions/sql-migration/src/wizard/targetSelectionPage.ts

```diff
@@ -1063,6 +1066,20 @@ export class TargetSelectionPage extends MigrationWizardPage {
 								sourceDatabaseName,
 								mappedSourceDatabaseName));
 					}
+
+					// Collation validation
+					if (sourceDatabaseCollation &&
```

Comment:
migrate to private function to do the check, i.e. isCollationSame.

also we should do case insentive compare using toLocaleLowerCase